paltas/Configs/config_handler.py

```diff
@@ -36,17 +37,25 @@ class ConfigHandler():
 		config_path (str): A path to the config file to parse.
 	"""
 
-	def __init__(self,config_path):
+	reseed_counter = 0
```

Comment:
Having this be a property of the class is a bit odd here. Would there be any time when we would want to access reseed_counter without initializing an instance of the class?

Reply:
OK, setting it explicitly as an instance variable in __init_ now